packages/nx-firebase/package.json

```
@@ -22,10 +22,9 @@
   "executors": "./executors.json",
   "peerDependencies": {
     "@nx/devkit": ">= 16.1.1",
+    "@nx/node": "^16.6.0",
     "@nx/workspace": ">= 16.1.1",
     "nx": ">= 16.1.1"
   },
```

After some testing, I can confirm this change is definitely needed to remove `@nx/node` as a package dependency altogether. However, I will need to add a couple of additional supporting changes in another PR.
```suggestion
  "peerDependencies": {
    "@nx/devkit": ">= 16.1.1",
    "@nx/workspace": ">= 16.1.1",
    "nx": ">= 16.1.1"
  },
```